src/js/profile/mobile/widget/TimePicker.js

```diff
@@ -252,7 +253,7 @@
 				ui.minuteInput.value = minutes;
 
 				spins.hour.value(hours);
-				spins.minute.value(minutes);
+				spins.minute.value((minutes < 10 ? "0" : "") + minutes);
```

Comment:
Do you really needs this?

Reply:
I refer to the TimePicker in wearable profile.
I will check it without it.

Reply:
I have to verify it

Reply:
@TomaszLukawskiSam,
You are right. It seems unnecessary.